### ISSUE TITLE
two version of ec2-nodes-plugin

### ISSUE THREAD

**Issue by SydOps** [rundeck-plugins/rundeck-ec2-nodes-plugin#38]
I installed two version rundeck-ec2-nodes-plugin (1.5 and 1.5.1) under $RDECK_BASE/libext

I think I should get two ec2 plugins, but I found ec2 plugin is totally disappeared in Rundeck.

**Comment by gschueler** [comment]
@SydOps I recently noticed this too.  the plugin needs to be rebuilt with additional metadata in the jar file.

since the metadata is wrong in 1.5.x, it is failing to correctly choose which one to use.  I will add the necessary data to the ec2 plugin. 

But, even then you would only see 1 plugin.  Rundeck will notice two versions, and select the "latest" version as the one to load. It would not load both. 

But if loading both versions is a feature you think rundeck should have, you might file it on the https://github.com/rundeck/rundeck project.

**Comment by SydOps** [comment]
Yes, I agree the idea that only select the "latest" version to be loaded. Otherwise, it will make confusion.

**Comment by SydOps** [comment]
I confirm this issue has been fixed, when do a testing on #37 

I have two versions:

```
rundeck-ec2-nodes-plugin-1.5.2-SNAPSHOT.jar
rundeck-ec2-nodes-plugin-1.5.1.jar
```

I can see ec2 plugin enabled in rundeck with latest new function `assume_role`(Assume Role ARN)
